packages/ui/src/showcases/DepositAndBorrowPage.stories.tsx

```diff
@@ -0,0 +1,410 @@
+      alignText: 'center',
+    },
+  }
+
+  const assetsToDepositColumns = {
+    ...defaultColumns,
+    action: {
+      header: '',
+      Cell: () => (
+        <Button
+          appearance="secondary"
+          variant="medium"
+          size="small"
+          onClick={() => setIsModalOpen('deposit')}
+        >
+          Deposit
+        </Button>
+      ),
+      width: 100,
+    },
+  }
+
+  const assetsToBorrowColumns = {
+    ...defaultColumns,
+    action: {
+      header: '',
+      Cell: () => (
+        <Button
+          appearance="secondary"
+          variant="medium"
+          size="small"
+          onClick={() => setIsModalOpen('borrow')}
+        >
+          Borrow
+        </Button>
+      ),
+      width: 100,
+    },
+  }
+
+  const defaultData = [
+    {
+      assets: 'ETH',
+      walletBalance: '1.250.000$',
+      APY: '3.5%',
+    },
+    {
+      assets: 'DAI',
+      walletBalance: '1.250.000$',
+      APY: '3.5%',
+    },
+    {
+      assets: 'POLKA',
+      walletBalance: '1.250.000$',
+      APY: '3.5%',
+    },
+    {
+      assets: 'BTC',
+      walletBalance: '1.250.000$',
+      APY: '3.5%',
+    },
+  ]
+  const DepositTitleBox = () => (
+    <StackLayout gap={2.125}>
+      <Text as="span" color="oxfordBlue" type="heading 2">
+        Your deposit
+      </Text>
+      <ColumnLayout center gap={1}>
+        <Text color="oxfordBlue" as="span" type="body 2">
+          Balance $
+          <Text as="span" color="oxfordBlue" bold>
+            1.00
+          </Text>
+        </Text>
+
+        <Text color="oxfordBlue" as="span" type="body 2">
+          Collateral $
+          <Text as="span" color="oxfordBlue" bold>
+            1.00
+          </Text>
+        </Text>
+      </ColumnLayout>
+    </StackLayout>
+  )
+
+  const BorrowTitleBox = () => (
+    <ColumnLayout className={stylings.tableHeaderWithProgressBar}>
+      <StackLayout gap={2.125}>
+        <Text as="span" color="oxfordBlue" type="heading 2">
+          Your borrow
+        </Text>
+
+        <Text color="oxfordBlue" type="body 2" as="span">
+          Balance $
+          <Text as="span" color="oxfordBlue" bold>
+            1.00
+          </Text>
+        </Text>
+      </StackLayout>
+      <ProgressBar
+        className={stylings.tableHeaderProgressBar}
+        title="Borrowed"
+        value={50}
+        showBottomLabel
+      />
+    </ColumnLayout>
+  )
+  const initialRowData = {
+    assets: 'Asset',
+    balance: '1.00',
+    APY: '0.50%',
+  }
+
+  const depositData = [initialRowData]
+  const borrowData = [initialRowData]
+
+  if (args.appendDepositRow) {
+    depositData.push(initialRowData)
+  }
+
+  if (args.appendBorrowRow) {
+    borrowData.push(initialRowData)
+  }
+
+  return (
+    <BrowserRouter>
+      <PageLayout {...nav}>
+        <PageLayout.Section>
+          <GridLayout>
+            <GridLayout.Column span={6}>
+              <StackLayout gap={1.5}>
+                <CardLayout>
```

Comment:
I guess that this is making double border, we should probably remove CardLayout where we have Table component, because table has border (and acts as CardLayout, atleast it should)